packages/i18n/package.json

```diff
@@ -41,12 +41,12 @@
   "dependencies": {
     "@babel/runtime-corejs3": "7.8.3",
     "@commercetools-frontend/sentry": "16.0.0",
+    "@commercetools-uikit/i18n": "^10.13.0",
```

Comment:
This was actually a bug, caught by the new tests. This dependency needs to be defined, as we did in #1306 and #1308